### DISCUSSION TITLE
How to set icon to heroicon

### DISCUSSION THREAD

**Discussion by kobe-f8** [jacksleight/statamic-bard-texstyle#55] [General]
Hello, I was wondering how you could set the icon to a heroicon icon, the documentations show it's a valid option but I haven't figured out how to get it to work.

I presumed I had to change the icon value to the name of the heroicon icon name but that doesn't work.

**Comment by jacksleight** [comment]
Hey. Ah yeah the docs should be clearer on this (will fix). Heroicons is just an example of using a custom SVG. To use them just copy the SVG code from the Heroicons site and paste it into the style's `icon` value.

**Reply by kobe-f8** [reply]
Awesome, I just realised this means I can use local svg's by using `file_get_contents`, how cool!
